src/graphql/outlier.rs

```diff
@@ -841,7 +850,7 @@ where
                         if node.distance < start || node.distance > end {
                             continue;
                         }
-                    } else if node.distance < start {
+                    } else if (((node.distance * 10.0).round() / 10.0) - start).abs() > 0.0 {
```

Comment:
When a user inputs a single number instead of a range, then this should pick up events with distance of `| d - user's number | < alpha`. I suggest 0.1 as alpha.